.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "scala",
+            "name": "Debug Hello World",
+            "request": "launch",
+            // "mainClass": "consoleapp",
+            // "buildTarget": "consoleapp",
+            "mainClass": "com.helloworld.Hello",
```

Comment:
Questo file è utilizzato davvero?

Per me OK includere file che aiutano per il testing in locale (anche se tendenzialmente direi di togliere i commenti) ma mi chiedo se lo stai davvero usando o se è rimasto lì, perché la main class ora dovrebbe essere `rps.Hello` e non `com.helloworld.Hello` (il prefisso è dato dal package), quindi qui sembra rimasto sbagliato.

Reply:
Meglio eliminarlo. Ho fatto qualche ricerca ma consigliano tutti di lasciare perdere questo approccio. Meglio usare `sbt`